### ISSUE TITLE
Playbook translation

### ISSUE THREAD

**Issue by palkan** [test-prof/docs-zh-cn#5]
Just added a brand new page: https://test-prof.evilmartians.io/playbook (https://github.com/test-prof/test-prof/blob/master/docs/playbook.md)

**Comment by xfyuan** [comment]
Ahh, just see the issue today. I'm a bit busy at the end of year, but I'll try to take the time to translate them. 🤝

**Comment by xfyuan** [comment]
@palkan I've compared the diff between the latest test-prof en vs zh_cn, and found there're several new en docs which does not exist in zh_cn. I think they should all be translated together with playbook.md, right?

Left is zh_cn in the below:
<img width="1209" alt="image" src="https://github.com/test-prof/docs-zh-cn/assets/5710838/388dfd6b-5a51-4878-9fee-78c7fc83e22d">

**Comment by xfyuan** [comment]
So I plan to add those 3 new files in a new PR:

- docs/playbook.md
- docs/profilers/memory_prof.md
- docs/profilers/ruby_profilers.md

and review all the old zh_cn version vs the latest en versions. I'll sync all translations of them to the newest in the PR also.

**Comment by palkan** [comment]
>  found there're several new en docs which does not exist in zh_cn

Yeah, I consilidated older ruby_prof.md and stack_prof.md into ruby_profilers.md (with Vernier added). And memory_prof.md is a new one.

Thanks! Can't wait to merge the updates 🙂
